app/assets/javascripts/application.js

```diff
@@ -8,7 +8,6 @@
 //= require vendor
 //= require jquery.extentions
 //= require jquery.multi-select
-//= require settings
```

Comment:
actually, re-reading this, it looks like settings was always in the application bundle, hence it didn't really trigger a full page reload....